main.py

```diff
@@ -219,16 +219,18 @@ async def list(message: Message):
     # Scrape all tracks in the message's channel and list them
     channel_media_attachments = await scrape_channel_media(message.channel)
 
-    message_to_send = "❤️‍🔥 AIGHT. IT'S BUSTY TIME ❤️‍🔥\n\n**Track Listing**"
+    embed_title = "❤🔥 AIGHT. IT'S BUSTY TIME ❤🔥"
+    embed_content = ""
 
-    for index, (author, filename, media_content_bytes) in enumerate(
+    for index, (author, filename, media_content_bytes, jump_url) in enumerate(
         channel_media_attachments
     ):
-        message_to_send += f"""
-{index+1}. {author.mention} - `{filename}`"""
+        embed_content += f"**{index+1}.** {author.mention} - [{discord.utils.escape_markdown(filename)}]({jump_url})\n"
```

Comment:
I added a separate file and message jump. I changed the format slightly, let me know if this seems fine to you.
See eb4c1705a43d8364827d876d00f6b8c4e8c69a9e 
![image](https://user-images.githubusercontent.com/6956898/149605972-431608c4-7252-4a97-ac54-e8ecb417f394.png)

Reply:
That looks good! And yes, the `↲jump` describes the action better, as well as being easier to press on mobile.

Speaking of mobile though (at least Discord on Android), it looks like Discord doesn't process markdown in embeds on mobile:

![Screenshot_20220115-152309](https://user-images.githubusercontent.com/1342360/149627191-023f6dae-c803-4374-bf8f-984fc6c353df.png)

which is a bit unfortunate. I'd say we merge this and take that to a separate issue though, and iterate from there.

Reply:
Interesting, yeah the same thing happens on my Android phone. I tested it on an iPad and it looks like this (I assume it's the same on iPhone), so I think this just an Android-only Discord bug.

![image](https://user-images.githubusercontent.com/6956898/149629023-055ce470-8330-4f3e-ab8c-c15583fa2d4a.png)

I submitted a bug report to Discord since it's an obscure enough bug they might not know about it yet (though who knows if they actually read those).

Reply:
Thanks for testing, and thanks for reporting to Discord. I've made an issue for us to track at https://github.com/anoadragon453/busty/issues/22, in case we want to try and figure out a workaround.